src/V3Sched.cpp

```diff
@@ -1032,13 +1058,19 @@ void schedule(AstNetlist* netlistp) {
 
     AstCFunc* const nbaFuncp = V3Order::order(
         netlistp, {&logicRegions.m_nba, &logicReplicas.m_nba}, trigToSenNba, "nba",
-        v3Global.opt.mtasks(), false, [](const AstVarScope*) { return nullptr; });
+        v3Global.opt.mtasks(), false, [&](const AstVarScope* vscp) -> AstSenTree* {
+            // If the var is written to after event controls, return their combined sentree
+            const auto it = timingKit.varToSenTreep.find(vscp);
+            if (it != timingKit.varToSenTreep.end()) return nbaTrigMap.at(it->second);
+            return nullptr;
+        });
```

Comment:
While this is the right thing to do (you need to do this to get combinational logic dependent on these variables to update), I think this should be in 'act'. Our original plan based on our earlier discussion was to have all resumptions performed before the 'act' region is evaluated, but I have not yet confirmed that is indeed what we do. 

I think you also need a separate trigger for "variables updated from delay-suspended processes" and set it when anything delayed was resumed.

Reply:
See main comment.

I may be confused, does `nba` logic (or logic currently treated as `nba`) dependant on suspendable stuff have to be resolved in `act`? If so, I guess we should color all logic dependant on suspendables to be in `act`?